WordPress/wordpress-playground
# Support PHP 8.3

## What is this PR doing?

Adds PHP 8.3 to Playground.

Closes https://github.com/WordPress/wordpress-playground/issues/820

<img width="1221" alt="CleanShot 2023-12-01 at 11 52 32@2x" src="https://github.com/WordPress/wordpress-playground/assets/205419/3008499a-f0d1-42a0-af3f-24d83e221329">

## Testing Instructions

1. Go to http://localhost:5400/website-server/ and switch the PHP version to 8.3.
2. Confirm that WordPress is functional and you can update the site in the site editor
3. Go to `/phpinfo.php`, confirm the php version it 8.3
4. Update settings to run with additional PHP extensions and networking support
5. Confirm that WordPress is functional, you can update the site in the site editor, and the plugin directory shows plugins and you can install them
6. Go to `/phpinfo.php`, confirm the php version it 8.3
7. In CLI, run `PHP=8.3 nx run php-wasm-cli:start -- -i` and confirm the PHP version presented on the screen is 8.3
